src/compiler/diagnosticMessages.json

```
@@ -2893,6 +2893,10 @@
         "category": "Error",
         "code": 2638
     },
+    "React components cannot include JSX namespace names": {

```

This is specific to attributes? If so:

```suggestion
    "React components cannot support namespaced attribute names.": {
```